listenbrainz/tests/integration/test_api.py

```diff
@@ -236,14 +236,6 @@ def test_unauthorized_submission(self):
         self.assert401(response)
         self.assertEqual(response.json['code'], 401)
 
-    def test_valid_single(self):
```

Comment:
Deleted accidentally?

Reply:
Yes, sorry for that. I add it back in new commits.